BUILDING.md

```diff
@@ -215,12 +215,13 @@ Consult previous versions of this document for older versions of Node.js:
 
 ## Building Node.js on supported platforms
 
+#### Installing Prerequisites
 The [bootstrapping guide](https://github.com/nodejs/node/blob/master/tools/bootstrap/README.md)
 explains how to install all prerequisites.
 
 ### Unix/macOS
 
-#### Prerequisites
+Prerequisites:
```

Comment:
Maybe we should keep "Prerequisites" under Unix/Mac OS section, so that user scan quickly link to.

Reply:
Sure! I will now link to both the 'Prerequisites' under Unix/macOS and the 'Prerequisites' under Windows.